server/engine/src/main/antlr4/org/eclipse/lsp/cobol/implicitDialects/sql/Db2SqlExecParser.g4

```diff
@@ -712,20 +713,22 @@ dbs_explain_stabilized: STABILIZED DYNAMIC QUERY STMTID (NUMERICLITERAL | dbs_in
 
 
 /*FETCH */
-dbs_fetch: FETCH (dbs_fetch_beforeafter | dbs_fetch_singlemulti);
-dbs_fetch_beforeafter: (BEFORE | AFTER) FROM? dbs_cursor_name;
-dbs_fetch_singlemulti: (INSENSITIVE | SENSITIVE)? (dbs_fetch_single | dbs_fetch_multi);
-dbs_fetch_single: (WITH CONTINUE)? dbs_fetch_rowpos? FROM? dbs_cursor_name dbs_fetch_singlerow?;
+//ref: https://www.ibm.com/docs/en/db2-for-zos/13?topic=statements-fetch
+dbs_fetch: FETCH (INSENSITIVE | SENSITIVE)? (WITH CONTINUE)? (BEFORE | AFTER | dbs_fetch_rowpos | dbs_fetch_rowsetpos)?
+            FROM? dbs_cursor_name dbs_fetch_single_multi_row;
+dbs_fetch_single_multi_row: (FOR (dbs_host_variable | dbs_integer_constant) ROWS)?  dbs_fetch_singlerow?;
```

Comment:
The multiple-row-fetch clause seems to be allowed only when one of the rowset options is specified.